models/attachment/src/index.ts

```diff
@@ -80,8 +82,19 @@ export class TSavedAttachments extends TPreference implements SavedAttachments {
   declare attachedTo: Ref<Attachment>
 }
 
+@Model(attachment.class.Drawing, core.class.Doc, DOMAIN_ATTACHMENT)
+export class TDrawing extends TDoc implements Drawing {
+  @Prop(TypeRef(core.class.Doc), getEmbeddedLabel('Parent'))
+  @Index(IndexKind.Indexed)
+  @Hidden()
+    parent!: Ref<Doc>
```

Comment:
If you are planning to add drawing not only for Attachment, then you need to add parent class attribute.

Reply:
Sure, added this as `parentClass` field